src/Model/Consignment/AbstractConsignment.php

```diff
@@ -652,38 +681,11 @@ public function isCdCountry()
      * @return bool
      * @todo move to hasCountry
```

Comment:
Wat is die todo? Kan dat weg?

Reply:
die stond er al en ik begreep niet wat er mee werd bedoeld, want 'hasCountry'. heeft toch een andere betekenis... van mij mag die weg overal in dat bestand. zal ik dat doen?

Reply:
Ah, laat maar staan, het is onduidelijk maar het gaat blijkbaar over de droom om `hasCountry` en `hasStreet` traits te maken en dat daarheen te verplaatsen, dat lijkt me een goed idee gezien dit bestand veel te groot is.

Reply:
@EdieLemoine dat was/is inderdaad de droom